Applications/PlazaDetails/PublishService.qmd

```diff
@@ -9,88 +9,82 @@ Users can publish their algorithm as a service in the openEO Algorithm Plaza. Th
 
 ##### Step 3: Additional Sections
 
-In addition to the basic information mentioned above, you can provide more details on your service by adding additional sections. The following sections are available:
+In addition to the basic information mentioned above, user can provide more details on their service by adding additional sections. The following sections are available:
 
-* Parameters: You can list all the parameters that should be fed to the service to execute it. You can specify the name, type, description, and default value for each parameter.
-* Usage example(Python code): You can provide a Python code example to demonstrate how to use your service. The code should give a complete example of how to use your service, including the use of the parameters and the namespace. 
-* Results: You can briefly describe your service's results and mention the output format of the result.
-* Cost Estimation: You can give a user an idea of the resource consumption and time required to run your service for a given input.
-* References: You can provide a list of references to publications, websites, or other resources relevant to your service. Provide details on resource consumption, processing time, and output format to help users effectively understand and utilize your service.
+* Parameters: A list of all the parameters that should be fed to the service to execute it. The name, type, description, and default value for each parameter should be specified.
+* Usage example(Python code): A Python code example to demonstrate how to use the service. The example should include the service's input parameters and the expected output.
+* Results: Service's results and it's output format should be mentioned.
+* Cost Estimation: An idea of the resource consumption and time required to run the service for a given input.
+* References: A list of references to publications, websites, or other resources relevant to the service should be provided. 
 
 ##### Step 4: Add labels
 
-You can add multiple labels to your service to help users find your service in the openEO algorithm plaza. The labels can be used to filter services within the marketplace and also give an idea of its category.
+Multiple labels can be added to the service to assist users in finding it within the openEO Algorithm Plaza. These labels serve as filters within the marketplace and also indicate the service's category.
 
 ##### Step 5: Select service visibility
 
-You can choose to make your service public or private. If you select public, your service will be visible to all users in the openEO algorithm plaza. If you select private, only you and your organization members will be able to see your service.
+The service can be designated as either public or private. If set to public, it will be visible to all users in the openEO Algorithm Plaza. On the other hand, if set to private, only the user and organization members will have access to the service.
```

Comment:
only the developer?

Consider making this distinction throughout the documentation to refer to the developer and users to improve the clarity of the explanation